systemtest/src/test/java/io/strimzi/systemtest/CustomResourceStatusST.java

```diff
@@ -250,6 +253,20 @@ void testKafkaConnectS2IStatus() throws InterruptedException {
         assertKafkaConnectS2IStatus(3, connectS2IUrl, connectS2IDeploymentConfigName);
     }
 
+    @Test
+    void testKafkaConnectorWithoutClusterConfig() {
+        // This test check NPE when connect cluster is not specified in labels
```

Comment:
Are you saying that the NotReady has a `NullPointerException` as its reason? If so we should fix that.

Reply:
No, it was already fixed and this test just check, that no NPE is in log during this test case.